defaults/main.yml

```diff
@@ -31,8 +31,8 @@ rhel7stig_workaround_for_ssg_benchmark: true
 # PLEASE NOTE: These work in coordination with the cat1, cat2, cat3 group variables. You must enable an entire group
 # in order for the variables below to take effect.
 # CAT 1 rules
-rhel_07_010010: true
-rhel_07_010020: true
+rhel_07_010010: false
```

Comment:
I don't think the consensus would be to have these disabled by default in the role. If you want to override the application of some of these fixes for specific machines then you can set these variables on a per-machine basis using group or host vars for the dev machines.

Reply:
On a vanilla install I haven't run into any issues, but when working in dev environments these 2 are the root cause of either system breaks or in cases of multi cluster installs I watched this take over 24 hours of run time to get through because of custom software utilizing ansible after RPM install to change ownership of files/directories.

Reply:
@jhultz - But our custom RPMs are the exception. Generally speaking this role would be run on a fresh RHEL installation, where that problem wouldn't be encountered. We can disable this on our own runs if executing after our RPMs get installed.